chromium_src/ui/views/controls/button/md_text_button.cc

```diff
@@ -284,8 +284,11 @@ void MdTextButton::UpdateTextColor() {
     return;
   }
 
-  auto colors = GetButtonColors();
-  SetTextColor(GetVisualState(), colors.text_color);
+  // To prevent set explicitly set color from here.
```

Comment:
Nit: Something is missing in the comment here - not clear what we are trying to prevent.

Reply:
Right, added more detailed comment. Thanks for checking!